src/views/Details/NFTAssetDetails.vue

```diff
@@ -15,6 +15,10 @@
         >
           <Star :nftAsset="nftAsset" :accountId="accountId" />
         </div>
+        <NFTQuantity
+          :quantity="nftAsset.amount"
+          :style="showFullscreen ? { top: 'calc(25% - 34px)' } : { bottom: 'calc(20% + 10px)' }"
```

Comment:
Is this calculation working in all cases: 
 :style="showFullscreen ? { top: 'calc(25% - 34px)' } : { bottom: 'calc(20% + 10px)' }" - looks like a hack